test/_e2e/config.go

```diff
@@ -8,7 +8,7 @@ import (
 // Intended to provide setup configuration for E2E tests
 const (
 	defaultRegistry        = "localhost:50000/user"
-	testTemplateRepository = "http://github.com/lance/test-templates.git" //nolint:varcheck,deadcode
+	testTemplateRepository = "http://github.com/boson-project/test-templates.git" //nolint:varcheck,deadcode
```

Comment:
Woah - how long has that been there?! Oops :)

Reply:
for months.. lol.. just notice now as well...